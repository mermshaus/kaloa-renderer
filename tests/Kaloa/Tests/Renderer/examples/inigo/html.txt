a[html]<div>Dies ist ein Test [b][/b]

foo bar &amp;</div>[/html]b
