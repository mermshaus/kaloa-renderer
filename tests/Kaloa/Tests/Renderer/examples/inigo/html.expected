<p>a</p>
<div>Dies ist ein Test [b][/b]

foo bar &amp;</div>

<p>b</p>
